Common/triggerSel.py

```diff
@@ -0,0 +1,51 @@
+import yaml
+class Triggers():
+    dict_legtypes = {"Electron":"Leg::e", "Muon":"Leg::mu", "Tau":"Leg::tau"}
+
+    def __init__(self, triggerFile):
+        with open(triggerFile, "r") as stream:
+            self.trigger_dict= yaml.safe_load(stream)
+        self.deltaR_matching = 0.4
+
+
+    def ApplyTriggers(self, df, isData = False):
+        hltBranches = []
+        df = df.Define(f"TrigObj_idx", f"CreateIndexes(TrigObj_pt.size())").Define("TrigObj_mass", "RVecF mass ; for(int i =0; i< TrigObj_pt.size(); i++){mass.push_back(0.);} return mass;")
+        df = df.Define(f"TrigObj_p4", f"GetP4(TrigObj_pt,TrigObj_eta,TrigObj_phi, TrigObj_mass, TrigObj_idx)")
```

Comment:
these lines should be in baseline together with initialisation of other idx and p4

Reply:
ok